octosql/handle_select.go

```diff
@@ -0,0 +1,32 @@
+package octosql
+
+import (
+	"context"
+	"github.com/cube2222/octosql/parser"
+	"github.com/cube2222/octosql/parser/sqlparser"
+	"github.com/cube2222/octosql/streaming/storage"
+	"github.com/dgraph-io/badger/v2"
+	"log"
+)
+
+// This function executes all SQL select statements
+func (e *OctosqlExecutor) handleSelectStatement(ctx context.Context, statement sqlparser.SelectStatement) error {
+	// Parse input statement
+	plan, err := parser.ParseNode(statement)
+	if err != nil {
+		log.Fatal("couldn't parse query: ", err)
```

Comment:
It would be better to return errors from this function, it will fatal above anyways.

Reply:
Fixed